runtime/src/tokio/telemetry.rs

```diff
@@ -50,7 +50,7 @@ pub fn init(
     let log_layer = if logging.json {
         log_layer.json().boxed()
     } else {
-        log_layer.pretty().boxed()
+        log_layer.compact().boxed()
```

Comment:
This led to some very odd formatting: https://docs.rs/tracing-subscriber/0.3.19/tracing_subscriber/fmt/format/index.html